e2e/app/page/conceptset-save-modal.ts

```diff
@@ -55,9 +54,6 @@ export default class ConceptsetSaveModal extends Modal {
       waitWhileLoading(this.page),
       this.waitUntilClose(),
     ]);
-
-    const conceptActionPage = new ConceptsetActionsPage(this.page);
-    await conceptActionPage.waitForLoad();
```

Comment:
Removed above code because there was a circular references:
`ConceptsetSearchPage -> ConceptsetSaveModal -> ConceptsetActionsPage -> ConceptsetSearchPage`